docs/implementations.rst

```diff
@@ -0,0 +1,72 @@
+=========================
+Saliency Implementations
+=========================
+
+Included with this toolkit are a number of implementations for the interfaces described in the previous section. Like those interfaces, these initial implementations focus of black-box visual saliency and are separated by use-case. Each implementation provides functionality for a different perturbation or heatmap generation algorithm specific to that use-case.
```

Comment:
I would recommend that each sentence be on a separate line, and use a 80-120 character width maximum.
This makes text blocks like this a bit easier to read in simpler text editors.
This also aids in simplifying change diffs: a spelling or sentence correction does not mark the whole "paragraph" as changed.

E.g. for this paragraph:
```suggestion
Included with this toolkit are a number of implementations for the interfaces
described in the previous section.
Like those interfaces, these initial implementations focus of black-box visual
saliency and are separated by use-case.
Each implementation provides functionality for a different perturbation or
heatmap generation algorithm specific to that use-case.use-case.
```

Reply:
Is it okay to do this with a fixup commit?